### ISSUE TITLE
Blacksmith Cost

### ISSUE THREAD

**Issue by Truetobi91** [Sh0wdown/UnofficialCrusaderPatch#469]
Since I was picking up on playing SHC I just noticed again, that the Blacksmith is the only weapon/armor crafter that costs 200g instead of 100g to build. I can't think of any reason for that as they are not even that efficient in producing gold for selling weapons (Wood is better for pure Gold gain and Armorer is faster than Blacksmith for the same gold value for swords/maces/armors. 

Could it be possible to have an option in the patch that reduces the gold cost for the Blacksmith to 100g?

**Comment by ByBurton** [comment]
Or alternatively make the weapons they produce a little more expensive (especially swords)?

**Comment by Truetobi91** [comment]
One thing to add: AI that have a high weapon production including Blacksmiths can run into the problem of delaying their Blacksmith until the end due to the higher gold costs, meaning everytime they have 100 gold they could build a different building than the Blacksmith.

**Comment by GRhin** [comment]
only reason I could see for the extra cost is that the developers wanted to discourage players going straight to the top melee units (macemen for speed, swordsmen for strength) too early in the match. their plan was likely to encourage players to start with spears (100 gold for a poleturner), then move into pikes (another 100 gold for armourer), then into swords if the game goes long enough to be worth it, and finally into knights. Each level of unit is a slightly bigger step up in investment than the level before.
This progression style does work for ranged units - start with bows, then move to crossbows. I suppose you could move from there into bows+maces, then into xbows+maces if you want to extend that path a little.
If this was their plan, I like the idea, however it didnt exactly work, but maybe byburton's idea of making swords more expensive would make a progression approach a bit more practical, though i think the cost of most weapons would need to go up, or at least the buy/sell prices be more different. Right now, you just buy the weapons you need untill you have the units to secure you while you build up eco to get the weapons production up, (often selling the weapons from the earlier weapons buildings like armourers and fletchers to pay for it)

**Comment by Rakso69** [comment]
Bro... It's because he can produce two different weapons both of which resources needed are iron, unlike for other workshops. It's not like the armourer who is limited only to metal armours.

**Comment by GRhin** [comment]
yes, but he cant do both at the same time, so that is not a valuable feature to building him. 
The fact that both need iron i dont see as relevant, because iron is also more expensive, the extra cost to make iron weapons is introduced in the price of iron (iron is slower to produce than wood as well, and theres the "Opportunity Cost" of not selling it as well). 
The only benefit within SHC for a blacksmith over a poleturner, is that the units that can be made with it are stronger or faster, depending which unit you are making, the downside being that its more expensive to keep running. The question is is that enough to justify doubling the cost of a blacksmith or not.

**Comment by Rakso69** [comment]
He can do both if you keep switching him. You pay extra gold for being capable of making both the weapons at will, instead of building 3 different iron-based workshops for each of the gear, but two. It's more expensive because it's iron, not wood, deal with it. Going by your logic an armourer should cost 10 wood 50 gold?
Maces are the fastest to make, swords second and metal armours are the slowest from the ironworks.
Using armourers you need like 4-5 of them for 2 iron mines. 3 mace-smiths would he enough on that situation, you pay 100-200 gold more, but 20-40 wood less.

**Comment by Rakso69** [comment]
The armourer is more expensive than the tanner.

**Comment by Rakso69** [comment]
Also bows vs xbows - bows are cheaper, same with spears vs pikes. Smith crafts cost the same.
Concluding: you can produce two weapons but they are no equal to eachother, meanwhile maces and sword pretty much are with only very little difference in terms of production speed.

**Comment by GRhin** [comment]
Sorry, my edit obviously came in after you started responding.
Im not sure which logic of my says the armourer should drop in cost, in fact, im not arguing any logic. I
In my Edit above i pointed out that the internal logic of weapons production is inconsistent, so theres no point in justfying the cost of blacksmiths by comparing them to the other buildings.
By the two trains of thought you have produced (iron being the basis of the weapons, and the fact that he makes two weapons), the tanner would need to be decreased in cost (cows vs armorer's iron, 1 armour vs poleturner and fletchers 2 weapons). - 10wood is not close to relative to the 100gold difference for blacksmith.
By your latest edit of iron craftables being produced faster. I dont see that as relevant, it is the value of the result compared to the cost that is relevant. Are swords and maces enough more profitable than crossbows to justify doubling the startup costs - i actually dont know, but I suspect that even with the slower production time, crossbows make you more gold than swords or maces. If that is correct then the value of a blacksmith should be about the same as a fletcher in my opinion. Even if not, you are limited to how many you can build by the number of iron you have close by, whereas crossbows you can buy more wood, or, on most maps, just build more woodcutters, so due to the relative benefits, blacksmiths are only worthwhile to a certain extent, and i see that as enough of a limitation to justify an equal cost.
And the latest message, relative prices of the weapons is also not relevant. If you are looking for sale value, you set it to the more profitable weapon. You aren't going to keep going into a building and changing the weapon every 30 seconds, you are just going to set it to your needs, and leave it until your needs change

**Comment by Rakso69** [comment]
Pointless to continue on discussing. All you see is "Irrelevant", or has "no point". So yeah, no point in continuing the discussion within your world where all what makes you happy is 2+2=4 without any depth within.

**Comment by GRhin** [comment]
lol, I do believe i was the one arguing against the 2+2=4 approach. But all good, this was a discussion on opinion anyways, You dont have to agree with me, i like to hear your views, but that doesnt mean i have to agree that your points are important to the discussion at hand.

**Comment by Rakso69** [comment]
Whatever. I just gave in points why they might cost 200 gold. Maybe it feels better on SH1 HD. You need 2 wood for a bow, and 4 wood for a crossbow there, which makes smiths 200 gold more senseworthy noticeable there, than on Crusader, I think the devs simply didn't change them, because they felt good with them as they are. Tho the wood ones seem needed to be produced faster than were on the previous game in order to match the european troops with the mercenary units gold-only cost I believe.

**Comment by Krarilotus** [comment]
@Rakso69 
> Maces are the fastest to make, swords second and metal armours are the slowest from the ironworks.

Just to clear out some missinformation: metal armours are actually the fastest from the iron workers
Speed of the weapon production is as follows:
Iron armour : Maces : Swords
94 : 79 : 74
Video proof:
https://www.youtube.com/watch?v=LbFN8VHx-iQ&t=604s&app=desktop

Thanks for the discussion.
I for my part would say, all the weapon production buildings are good as they are besides the fletcher and the iron armourer, which should cost as much as a apothecary in my opinion, so 150 gold + 20 wood
Giving swords a different iron ratio might also be a good option: just let them be out of 2 Iron and double buy and sell price.

Honestly, i wouldnt bother with these, the main argument was already made @Truetobi91, giving the AI faster buildup for blacksmiths by lowering cost to 100 gold. I dont think thats necessarily a good thing. There is no situation the AI should rush sword/mace production anyways, so keeping it more expensive encourages the AI to play more reasonable and safe.

Peace :dancer:
